redis/Dockerfile

```diff
@@ -1,18 +1,13 @@
-FROM redisfab/redisgears:master-x64-bionic as gears
-FROM redisfab/redisai:master-cpu-x64-bionic as ai
+FROM redislabs/redisai:latest as ai
```

Comment:
if we do not set different runtime params for the module, why do we need a docker file?